config/routes.rb

```diff
@@ -29,6 +29,9 @@
     namespace :exports do
       resources :organisations, only: [:show] do
         get "transactions", on: :member
+        get "programme_activities", on: :member
```

Comment:
Is it worth adding 'iati' to these, jsut to be super clear and keep them in a namespace of sorts?

Reply:
Good plan 👍 I've also namespaced the model, just to be consistent